### PR TITLE
Fix faucet to use correct endpoint and use updated encryption

### LINKED ISSUE #8479: Faucet plugin build is not published as npm
### Expected behavior
`./build` folder should be included in the 

### Actual behavior
In the package downloaded from the npm does not include the build folder

### Steps to reproduce
npm i @liskhq/lisk-framework-faucet-plugin

### Which version(s) does this affect? (Environment, OS, etc...)
6.0.0-beta.2

### LINKED ISSUE #8480: Faucet use legacy endpoint and key path
### Expected behavior
* Faucet should use updated endpoint
* Faucet should not use passphrase with legacy key path

### Actual behavior
* `faucet:fundTokens` is called instead of `faucet_fundTokens`
* faucet plugin is using legacy key path

### Which version(s) does this affect? (Environment, OS, etc...)
6.0.0-beta.2

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #8479 and resolves #8480 

### How was it solved?

- Update to use `privateKey` instead of `passphrase`
- Update endpoint name
- Check build is included through `npm publish --dry-run`

### How was it tested?

Run faucet locally and check if it sends the transaction correctly

### CODE REVIEW

**Review by ManuGowda — APPROVED** [review]
LGTM.

can you also update the copyright to 2023?

**Review by ManuGowda — APPROVED** [review]
LGTM.

can you also update the copyright to 2023?

**Review by ishantiw — APPROVED** [review]

File: framework-plugins/lisk-framework-faucet-plugin/src/plugin/schemas.ts [hunk 1]
```diff
@@ -60,11 +65,11 @@ export const configSchema = {
 			description: 'The re-captcha site key',
 		},
 	},
-	required: ['encryptedPrivateKey', 'captchaSecretkey', 'captchaSitekey'],
+	required: ['tokenID', 'encryptedPrivateKey', 'captchaSecretkey', 'captchaSitekey'],
 	default: {
 		port: 4004,
 		host: '127.0.0.1',
-		applicationUrl: 'ws://localhost:8080/ws',
+		applicationUrl: 'ws://localhost:7887/rpc-ws',
```
Comment (ManuGowda) [inline framework-plugins/lisk-framework-faucet-plugin/src/plugin/schemas.ts:72]: will this port be a problem when its testnet? if we have different port for beta and testnet then its best to read it dynamically?

Comment (shuse2) [reply framework-plugins/lisk-framework-faucet-plugin/src/plugin/schemas.ts:72]: this is default value, so it can be set from config

**Pull request merged**